x-pack/plugins/notifications/server/services/email_service.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface IEmailService {
```

Comment:
NIT: misleading naming. Rename this file `types.ts` and the `email.ts` file (containing the service) `email_service.ts`.